docs/user/aws/limits.md

```diff
@@ -0,0 +1,74 @@
+# AWS Limits
+
+You can find a comprehensive list of the default AWS service limits published here:
+
+[AWS Service Limits][service-limits]
+
+Below, we’ll identify OpenShift cluster needs and how those impact some of those limits.
+
+## S3
+
+There is a limit of 100 S3 buckets per account. The installation creates a bucket temporarily. Also, the registry 
```

Comment:
"a limit" -> "a default limit" or "an initial limit" or some such?

Reply:
I do not think AWS is flexible on the number of S3 buckets. This, historically, has been one of their harder limits. I can see we have more in openshift-dev, so I'll change this up a little bit. Thanks!